metadata.go

```diff
@@ -80,12 +82,36 @@ var debianLicenseText = map[string]string{
 `,
 }
 
+var githubRegexp = regexp.MustCompile(`github\.com/([\w._-]+/[\w._-]+)`)
```

Comment:
After I first committed this, and try to test with https://pault.ag/go/config?go-get=1 it matches `https://github.com/paultag/go-config"`